app/views/support_interface/application_forms/immigration_status/edit.html.erb

```diff
@@ -1,51 +1,48 @@
+          <% end %>
+
           <%= f.govuk_radio_button :immigration_status, :indefinite_leave_to_remain_in_the_uk, label: { text: t('indefinite_leave_to_remain_in_the_uk', scope:) } %>
           <%= f.govuk_radio_button :immigration_status, :student_visa, label: { text: t('student_visa', scope:) } %>
           <%= f.govuk_radio_button :immigration_status, :graduate_visa, label: { text: t('graduate_visa', scope:) } %>
           <%= f.govuk_radio_button :immigration_status, :skilled_worker_visa, label: { text: t('skilled_worker_visa', scope:) } %>
           <%= f.govuk_radio_button :immigration_status, :dependent_on_partners_or_parents_visa, label: { text: t('dependent_on_partners_or_parents_visa', scope:) } %>
           <%= f.govuk_radio_button :immigration_status, :family_visa, label: { text: t('family_visa', scope:) } %>
-          <%= f.govuk_radio_button :immigration_status, :british_national_overseas_visa, label: { text: t('british_national_overseas_visa', scope:) } %>
+
+          <% unless @immigration_status_form.eu_nationality? %>
+            <%= f.govuk_radio_button :immigration_status, :british_national_overseas_visa, label: { text: t('british_national_overseas_visa', scope:) } %>
+          <% end %>
           <%= f.govuk_radio_button :immigration_status, :uk_ancestry_visa, label: { text: t('uk_ancestry_visa', scope:) } %>
           <%= f.govuk_radio_button :immigration_status, :high_potential_individual_visa, label: { text: t('high_potential_individual_visa', scope:) } %>
           <%= f.govuk_radio_button :immigration_status, :youth_mobility_scheme, label: { text: t('youth_mobility_scheme', scope:) } %>
-          <%= f.govuk_radio_button :immigration_status, :india_young_professionals_scheme_visa, label: { text: t('india_young_professionals_scheme_visa', scope:) } %>
-          <%= f.govuk_radio_button :immigration_status, :ukraine_family_scheme_or_ukraine_sponsorship_scheme_visa, label: { text: t('ukraine_family_scheme_or_ukraine_sponsorship_scheme_visa', scope:) } %>
-          <%= f.govuk_radio_button :immigration_status, :afghan_citizens_resettlement_scheme_or_afghan_relocations_and_assistance_policy, label: { text: t('afghan_citizens_resettlement_scheme_or_afghan_relocations_and_assistance_policy', scope:) } %>
+
+          <% unless @immigration_status_form.eu_nationality? %>
+            <%= f.govuk_radio_button :immigration_status, :india_young_professionals_scheme_visa, label: { text: t('india_young_professionals_scheme_visa', scope:) } %>
+            <%= f.govuk_radio_button :immigration_status, :ukraine_family_scheme_or_ukraine_sponsorship_scheme_visa, label: { text: t('ukraine_family_scheme_or_ukraine_sponsorship_scheme_visa', scope:) } %>
+            <%= f.govuk_radio_button :immigration_status, :afghan_citizens_resettlement_scheme_or_afghan_relocations_and_assistance_policy, label: { text: t('afghan_citizens_resettlement_scheme_or_afghan_relocations_and_assistance_policy', scope:) } %>
```

Comment:
The label for 'Afghan citizens resettlement scheme or Afghan Relocations and Assistance Policy' should be
 'Afghan Citizens Resettlement Scheme (ACRS) or Afghan Relocations and Assistance Policy (ARAP)'